### PR TITLE
Issue #SB-2724 feat: Breadcrumbs and tenant service

### PR DESCRIPTION
**Pull request by shakkeel1330**

### CODE REVIEW

File: src/app/migration/src/package.json [hunk 1]
```diff
@@ -45,9 +45,13 @@
     "ng2-charts": "^1.6.0",
     "ng2-izitoast": "^1.0.1",
     "ng2-semantic-ui": "^0.9.6",
+    "ngx-md": "^3.1.1",
+    "ngx-order-pipe": "^2.0.1",
     "ngx-slick": "^0.1.3",
+    "pagedown-core": "git+https://github.com/edrohler/pagedown-core.git",
```
Comment (anoophm) [inline src/app/migration/src/package.json]: we dont need this

File: src/app/migration/src/src/app/modules/announcement/announcement-routing.module.ts [hunk 2]
```diff
@@ -7,13 +7,15 @@ import { OutboxComponent, DeleteComponent, DetailsComponent, DetailsPopupCompone
 const routes: Routes = [
   {
     path: 'announcement/outbox/:pageNumber', component: OutboxComponent, canActivate: [AuthGuard],
+    data: [{label: 'Home', url: '/home'}, {label: 'Profile', url: '/profile'}, {label: 'My Announcements', url: ''}],
```
Comment (anoophm) [inline src/app/migration/src/src/app/modules/announcement/announcement-routing.module.ts]: {label: 'My Announcements', url: ''} should not be clickable

File: src/app/migration/src/src/app/modules/core/components/breadcrumbs/breadcrumbs.component.html [hunk 3]
```diff
@@ -0,0 +1,15 @@
+<div class="ui margin-bottom-10 margin-top-15 breadCrumbsBackground" *ngIf="breadCrumbsData" >
+  <div class="ui grid">
+      <div class="one wide column"></div>
+      <div class="ten wide column">
+          <div class="breadcrumppadding ui padded breadcrumb">
+              <span *ngFor="let item of breadCrumbsData; last as isLast">
+                  <a *ngIf="!isLast" (click)="openLink(item.url)"  class="breadcrumb-fontColor section">{{item.label}}</a>
```
Comment (anoophm) [inline src/app/migration/src/src/app/modules/core/components/breadcrumbs/breadcrumbs.component.html]: for the last item in breadcrumbs array navigation should not happen

File: src/app/migration/src/src/app/modules/core/components/main-header/main-header.component.ts [hunk 4]
```diff
@@ -1,6 +1,8 @@
+import { TenantService } from './../../services/tenant/tenant.service';
```
Comment (anoophm) [inline src/app/migration/src/src/app/modules/core/components/main-header/main-header.component.ts]: it should be import { TenantService } from './../../services'

File: src/app/migration/src/src/app/modules/core/components/main-header/main-header.component.ts [hunk 5]
```diff
@@ -79,13 +86,21 @@ export class MainHeaderComponent implements OnInit {
       (user: IUserData) => {
           if (user && !user.err) {
             this.userProfile = user.userProfile;
+            const rootOrg = this.userProfile.rootOrg;
+            this.tenantService.getOrgDetails(rootOrg.slug).subscribe((data) => {
+              this.orgLogo = data.result.logo;
```
Comment (anoophm) [inline src/app/migration/src/src/app/modules/core/components/main-header/main-header.component.ts]: we get favicon should also be added

File: src/app/migration/src/src/app/modules/core/services/tenant/tenant.service.ts [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+
+  /**
+   * API call to gather organization details.
+   * @param orgSlug Organization details passed from main-header component.
+   */
+  public getOrgDetails(orgSlug) {
+    const option = {
+      url: `${this.config.urlConFig.URLS.TENANT.INFO + '/'}${orgSlug}`,
+      data: {
+        slug: orgSlug
```
Comment (anoophm) [inline src/app/migration/src/src/app/modules/core/services/tenant/tenant.service.ts]: we dont need to send data for get

File: src/app/migration/src/src/app/modules/core/services/tenant/tenant.service.ts [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+      }
+    };
+    const orgDetails = this.get(option);
```
Comment (anoophm) [inline src/app/migration/src/src/app/modules/core/services/tenant/tenant.service.ts]: return this.get(option);

File: src/app/migration/src/src/typings.d.ts [hunk 8]
```diff
@@ -8,4 +8,6 @@ declare module '*.json' {
   const sample: any;
   export default sample;
 }
-declare var EkTelemetry: any;
\ No newline at end of file
+
+declare const Markdown;
```
Comment (anoophm) [inline src/app/migration/src/src/typings.d.ts]: why we need this

File: src/app/migration/src/src/app/modules/core/components/breadcrumbs/breadcrumbs.component.html (hunk 3 above)
Comment (shakkeel1330) [reply src/app/migration/src/src/app/modules/core/components/breadcrumbs/breadcrumbs.component.html]: '!isLast' makes sure the last breadcrumb is omitted.

**Pull request closed without merging**